ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyChannelInitializer.kt

```diff
@@ -132,6 +141,19 @@ public class NettyChannelInitializer(
         }
     }
 
+    private fun EngineSSLConnectorConfig.hasTrustStore() = trustStore != null || trustStorePath != null
+
+    private fun EngineSSLConnectorConfig.trustManagerFactory(): TrustManagerFactory? {
+        val trustStore = trustStore ?: trustStorePath?.let { file ->
+            FileInputStream(file).use { fis ->
+                KeyStore.getInstance(KeyStore.getDefaultType()).also { it.load(fis, null) }
```

Comment:
In other places, we simply use `"JKS"` constant while here we are querying JCA for this value, is it intended?

Reply:
No, and fixed